importer/DeHeDe.py

```diff
@@ -0,0 +1,90 @@
+from Monument import Monument, Dataset
+import importer_utils as utils
+import importer as importer
+
+
+class DeHeDe(Monument):
+
+    def set_location(self):
+        if self.has_non_empty_attribute("ortsteil"):
+            location_q = utils.q_from_wikipedia("de", self.ortsteil)
+            if location_q:
+                self.add_statement("location", location_q)
+
+    def set_adm_location(self):
+        city_q = utils.q_from_wikipedia("de", self.stadt)
+        self.add_statement("located_adm", city_q)
```

Comment:
would be good to check that you got a value for `city_q` and if not add to report instead

Reply:
Done.